src/solvers/dg/indicators.jl

```diff
@@ -45,7 +50,12 @@ function IndicatorHennemannGassner(semi::AbstractSemidiscretization;
                                    alpha_max=0.5,
                                    alpha_min=0.001,
                                    alpha_smooth=true,
-                                   variable=first)
+                                   variable=nothing)
```

Comment:
Same argument as above, and also for the remaining indicators...

Reply:
Thanks! Fixed in b5266b7.